.changeset/late-books-greet.md

```diff
@@ -0,0 +1,21 @@
+---
+'example-app': patch
+'@backstage/create-app': patch
+'@backstage/plugin-catalog-import': minor
+---
+
+The plugin has been refactored and is now based on a configurable state machine of 'analyze', 'prepare', 'review' & 'finish'.
```

Comment:
And the example should use `CatalogImportPage` since it has been migrated to a new name in the meanwhile... I did it in the code and readme, but missed the changeset.